content/md/en/docs/tutorials/work-with-pallets/use-macros-in-a-custom-pallet.md

```diff
@@ -123,6 +123,7 @@ Therefore, the first step is to remove some files and content from the files in
 
       #[pallet::pallet]
       #[pallet::generate_store(pub(super) trait Store)]
+      #[pallet::without_storage_info]
```

Comment:
I would prefer we put `#[pallet::unbounded]` above the storage which is bounded, or even better, make the tutorial use only bounded storage.

Reply:
I see you added `unbounded` below, which means this is not needed.